### PR TITLE
Use IOUtils#toString instead of own implementation

### PR DESCRIPTION
**Pull request by RoiEXLab**
Just a small refactoring change

### CODE REVIEW

File: src/main/java/games/strategy/util/Util.java [hunk 1]
```diff
@@ -122,16 +120,6 @@ public static String createUniqueTimeStamp() {
     });
   }
 
-  public static String getStringFromInputStream(InputStream in) {
-    StringBuilder builder = new StringBuilder();
-    try (Scanner scanner = new Scanner(in)) {
-      while (scanner.hasNextLine()) {
-        builder.append(scanner.nextLine()).append("\n");
```
Comment (ron-murhammer) [inline src/main/java/games/strategy/util/Util.java]: Are you sure that the Apache util also appends an end line in the string like this? Just want to make sure the string format is the same.

Comment (RoiEXLab) [reply src/main/java/games/strategy/util/Util.java]: No I'm not, but I believe apache is actually more accurate, so if \r\n is provided as line break \r\n will be used.
Shouldn't change any behaviour though: The old apache httpclient had a "getString" method, which provided the direct string as well, they probably used the same algorithm as in their IOUtils lib, so it worked earlier, it is going to work now!

**Comment by ssoloff** [comment]
You may want to consider using [`EntityUtils#toString()`](https://hc.apache.org/httpcomponents-core-ga/httpcore/apidocs/org/apache/http/util/EntityUtils.html#toString(org.apache.http.HttpEntity)) instead of `IOUtils#toString()`.  `EntityUtils` will use the `Content-Encoding` response header, if present.  There's no guarantee the entity will always be encoded using UTF-8.

**Comment by RoiEXLab** [comment]
@ssoloff Good Point, I'll submit a follow-up PR later this day

**Pull request merged**